app/assets/javascripts/admin/addon/components/admin-plugin-config-page.gjs

```diff
@@ -26,6 +30,30 @@ export default class AdminPluginConfigPage extends Component {
         <AdminPluginConfigTopNav />
       {{/if}}
 
+      <DBreadcrumbsContainer />
```

Comment:
It's weirding me out a bit that the container is self-closing, and the items are "outside". Could you explain what's going on? 🙂

Reply:
Yup -- so what is happening here is that we can have multiple containers on the page. These containers render _all items_ that have been registered. So the flow looks like this:

1. Container is rendered
2. Modifier on the container registers the container with DBreadcrumbsService
3. We look at the service.containers on each item to render each item into every container

The main benefit of this is that you can register a container on a top-level route/page and then each item added lower down in the tree automatically ends up in the container, without having to pass down e.g. a reference to the container down the entire render tree.

Reply:
Makes sense. Thanks! 🙏